tests/special_tests.sh

```diff
@@ -78,6 +78,12 @@ if [ $? -eq 0 ]; then
     report+="Ran\ttests/utilities/test_warnings.py\n"
 fi
 
+# test that a user can manually launch individual processes
+args="--trainer.gpus 2 --trainer.accelerator ddp --trainer.max_epochs=1 --data.batch_size=32 --trainer.limit_train_batches=2 --trainer.limit_val_batches=2"
```

Comment:
For the purpose of this test, this should be enough

```suggestion
args="--trainer.gpus 2 --trainer.accelerator ddp --trainer.fast_dev_run=True
```

Reply:
I took this from the existing tests